tests/vm_test.go

```diff
@@ -2062,16 +2063,11 @@ func createRunningVM(virtClient kubecli.KubevirtClient, template *v1.VirtualMach
 
 func startVM(virtClient kubecli.KubevirtClient, vm *v1.VirtualMachine) *v1.VirtualMachine {
 	By("Starting the VirtualMachine")
-	err := tests.RetryWithMetadataIfModified(vm.ObjectMeta, func(meta k8smetav1.ObjectMeta) error {
-		vm, err := virtClient.VirtualMachine(meta.Namespace).Get(context.Background(), meta.Name, k8smetav1.GetOptions{})
-		Expect(err).ToNot(HaveOccurred())
-		vm.Spec.Running = nil
-		runStrategyAlways := v1.RunStrategyAlways
-		vm.Spec.RunStrategy = &runStrategyAlways
-		_, err = virtClient.VirtualMachine(meta.Namespace).Update(context.Background(), vm, metav1.UpdateOptions{})
-		return err
-	})
-	Expect(err).ToNot(HaveOccurred())
+	patchData, err := createVMRunStrategyPatch(v1.RunStrategyAlways)
+	Expect(err).ShouldNot(HaveOccurred())
+
+	_, err = kubevirt.Client().VirtualMachine(vm.Namespace).Patch(context.Background(), vm.Name, types.JSONPatchType, patchData, metav1.PatchOptions{})
+	Expect(err).ShouldNot(HaveOccurred())
```

Comment:
I think in most places, we use `NotTo` or ToNot`, `Should*` is rarely used or not at all.
Same all over the code here.

Reply:
Yep thanks!